biodatasets/medhop/medhop.py

```diff
@@ -0,0 +1,232 @@
+                            "s": datasets.Value("string"),
+                            "o": datasets.Value("string"),
+                        }
+                    ],
+                    "snippets": [
+                        {
+                            "offsetInBeginSection": datasets.Value("int32"),
+                            "offsetInEndSection": datasets.Value("int32"),
+                            "text": datasets.Value("string"),
+                            "beginSection": datasets.Value("string"),
+                            "endSection": datasets.Value("string"),
+                            "document": datasets.Value("string"),
+                        }
+                    ],
+                }
+            )
+
+        # simplified schema for QA tasks
+        elif self.config.schema == "bigbio_qa":
+
+            features = schemas.qa_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+
+        my_urls = _URLs[self.config.schema]
+        data_dir = dl_manager.download_and_extract(my_urls)
+        data_dir += "/qangaroo_v1.1/medhop/"
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "train.json"),
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "dev.json"),
+                    "split": "validation",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath, split):
+        """Yields examples as (key, example) tuples."""
+
+        if self.config.schema == "source":
+
+            with open(filepath, encoding="utf-8") as file:
+
+                uid = 0
+
+                data = json.load(file)
+
+                for i, record in enumerate(data):
+
+                    record["type"] = "summary"
+
+                    yield i, {
+                        "id": record["id"],
+                        "type": record["type"],
+                        "body": record["query"],
+                        "documents": record["supports"],
+                        "concepts": record["concepts"] if "concepts" in record else [],
+                        "triples": record["triples"] if "triples" in record else [],
+                        "ideal_answer": [record["answer"]],
+                        "exact_answer": [record["answer"]],
+                        "snippets": record["snippets"] if "snippets" in record else [],
+                    }
```

Comment:
the source schema should capture the dataset with as little change as possible. can you comment on why the source_schema doesn't match these yields? For example, 
* why is `record["type"]` set to summary for each sample? 
* why is `record["answer"]` used for both `ideal_answer` and `exact_answer` ?

Reply:
The `type` isn't available in the JSON file. We only have the following fields :

```json
{
  "candidates": [
   "DB00072",
   "DB02901",
   "DB04844"
  ],
  "answer": "DB00072",
  "id": "MH_train_0",
  "supports": [
   "Induction of apoptosis of Beta cells ...",
   "DB00820 , a further innovation in the treatment ..."
  ],
  "query": "interacts_with DB00773?"
}
```

Since it doesn't have any `type`, I define it to `summary` by default.

For `exact_answer`, I made a mistake, since I have to wrap it in a `tuple` according to the `BioASQ` example :

https://github.com/bigscience-workshop/biomedical/blob/94237784b0cd801f691a602417bf14f4ac4dcf41/examples/bioasq.py#L522-L541

Reply:
for the source schema you should not copy any other dataset schema. the idea is to represent the data as it exists in this dataset as closely as possible.

Reply:
Okay, I understand now! I haven't experienced this difference in previous corpora, since they are based on the `BioC` format.

Reply:
Commit b821fc649c3371cff57713a5ee53e7483708f11d

The new source format give an output like this. Identical to the raw data : 

```json
{
   "id":"MH_train_1",
   "candidates":[
      "DB00294",
      "DB00313",
      "DB00588",
      "DB00755",
      "DB00783",
      "DB02546",
      "DB02901",
      "DB06822",
      "DB08910"
   ],
   "answer":"DB06822",
   "supports":[
      "Prolonged treatment with bicalutamide induces androgen receptor overexpression and androgen hypersensitivity . BACKGROUND : Various hormone refractory prostate cancer cell models have been established with androgen depletion and have helped to clarify the mechanism for the transition into androgen-depletion independent status . However , the mechanism of bicalutamide resistance remains unclear because few cell models have been generated . METHODS : We generated a bicalutamide-resistant subline , LNCaP- O43633 , from LNCaP after prolonged treatment with bicalutamide . Androgen and/or bicalutamide responsiveness for proliferation and prostate-specific antigen ( PSA ) secretion were examined in vitro and in vivo . DB00624 and dihydrotestosterone ( DB02901 ) levels in xenografted tumors were analyzed by liquid chromatography-tandem mass spectrometry . P10275 ( AR ) gene mutation and amplification and AR and pAR(210) expression were determined . RESULTS : LNCaP- O43633 did not grow in an androgen-depleted medium and proliferation was stimulated in a tenfold lower concentration of androgen than that of LNCaP . LNCaP- O43633 grew in castrated male mice , and the DB02901 level in grafted LNCaP- O43633 tumors was 7.7-fold lower than in LNCaP tumors . DB01128 stimulated LNCaP- O43633 proliferation and PSA secretion in vitro and the antitumor activity of bicalutamide against LNCaP- O43633 was weaker than that of LNCaP in vivo . Additional AR mutation and AR gene amplification were not detected in LNCaP- O43633 , but AR and pAR(210) expression and PSA secretion in LNCaP- O43633 were higher than in LNCaP . CONCLUSIONS : DB01128 -resistant LNCaP- O43633 exhibited AR overexpression and hypersensitivity to low levels of androgen . Our data suggests that AR overexpression is a significant mechanism of bicalutamide resistance similar to resistance from chronic androgen depletion . In addition , pAR(210) overexpression could be a potential mechanism for hypersensitivity to low androgen in LNCaP- O43633 .",
      "Reconstruction and functional analysis of altered molecular pathways in human atherosclerotic arteries . BACKGROUND : Atherosclerosis affects aorta , coronary , carotid , and iliac arteries most frequently than any other body vessel . There may be common molecular pathways sustaining this process . Plaque presence and diffusion is revealed by circulating factors that can mediate systemic reaction leading to plaque rupture and thrombosis . RESULTS : We used DNA microarrays and meta-analysis to study how the presence of calcified plaque modifies human coronary and carotid gene expression . We identified a series of potential human atherogenic genes that are integrated in functional networks involved in atherosclerosis . Caveolae and JAK/ P35610 pathways , and P06702 / P05109 interacting proteins are certainly involved in the development of vascular disease . We found that the system of caveolae is directly connected with genes that respond to hormone receptors , and indirectly with the apoptosis pathway . Cytokines , chemokines and growth factors released in the blood flux were investigated in parallel . High levels of RANTES , IL-1ra , MIP-1 alpha , MIP-1 beta , P60568 , P05112 , P05113 , P05231 , P13232 , Q16552 , DB00102 , P15692 and P01579 were found in plasma of atherosclerotic patients and might also be integrated in the molecular networks underlying atherosclerotic modifications of these vessels . CONCLUSION : The pattern of cytokine and P06702 / P05109 up-regulation characterizes atherosclerosis as a proinflammatory disorder . Activation of the JAK/ P35610 pathway is confirmed by the up-regulation of P05231 , P42224 , Q00978 and Q13651 genes in coronary and carotid plaques . The functional network constructed in our research is an evidence of the central role of P35610 protein and the caveolae system to contribute to preserve the plaque . Moreover , Cav-1 is involved in SMC differentiation and dyslipidemia confirming the importance of lipid homeostasis in the atherosclerotic phenotype .",
      "Targeting Q01196 / Q06455 -histone deacetylase repressor complex : a novel mechanism for valproic acid-mediated gene expression and cellular differentiation in Q01196 / Q06455 -positive acute myeloid leukemia cells . In t(8;21) acute myeloid leukemia ( AML ) , the Q01196 / Q06455 fusion protein promotes leukemogenesis by recruiting class I histone deacetylase ( HDAC ) -containing repressor complex to the promoter of Q01196 target genes . Valproic acid ( DB00313 ) , a commonly used antiseizure and mood stabilizer drug , has been shown to cause growth arrest and induce differentiation of malignant cells via HDAC inhibition . DB00313 causes selective proteasomal degradation of Q92769 but not other class I HDACs ( i.e. , HDAC 1 , 3 , and 8 ) . Therefore , we raised the question of whether this drug can effectively target the leukemogenic activity of the Q01196 / Q06455 fusion protein that also recruits Q13547 , a key regulator of normal and aberrant histone acetylation . We report here that DB00313 treatment disrupts the Q01196 / Q06455 - Q13547 physical interaction , stimulates the global dissociation of Q01196 / Q06455 - Q13547 complex from the promoter of Q01196 / Q06455 target genes , and induces relocation of both Q01196 / Q06455 and Q13547 protein from nuclear to perinuclear region . Furthermore , we show that mechanistically these effects associate with a significant inhibition of HDAC activity , histone H3 and H4 hyperacetylation , and recruitment of RNA polymerase II , leading to transcriptional reactivation of target genes ( i.e. , P08700 ) otherwise silenced by Q01196 / Q06455 fusion protein . Ultimately , these pharmacological effects resulted in significant antileukemic activity mediated by partial cell differentiation and caspase-dependent apoptosis . Taken together , these data support the notion that DB00313 might effectively target Q01196 / Q06455 -driven leukemogenesis through disruption of aberrant Q13547 function and that DB00313 should be integrated in novel therapeutic approaches for Q01196 / Q06455 -positive AML .",
      "Glucocorticoid-induced surface expression of annexin 1 blocks beta2-integrin adhesion of human eosinophils to intercellular adhesion molecule 1 surrogate protein . BACKGROUND : Glucocorticoids attenuate the population of eosinophils and T lymphocytes in asthmatic airways . The decrease in airway eosinophilia is caused both by accelerated cell death and by induction of blockade of integrin adhesion . In this study , we examined the hypothesis that annexin 1 surface expression , which is upregulated by the glucocorticoid receptor , prevents integrin adhesion essential to cell migration by blocking intracellular translocation of cytosolic group IV phospholipase A2 ( P47712 ) . OBJECTIVE : To examine the relationship of the glucocorticoid on annexin 1 expression and the effect of blockade of annexin 1 activity on adhesion of human eosinophils in vitro . To determine the relationship between annexin 1surface expression and nuclear membrane translocation of P47712 . METHODS : Eosinophils isolated from human peripheral blood were pretreated with fluticasone propionate ( FP ) , and beta2-integrin adhesion was measured after stimulation with P05113 or eotaxin . Effects of FP on P47712 expression , phosphorylation , and translocation were determined . The role of annexin 1 was examined by using annexin 1 blocking antibody and/or mimetic peptides . RESULTS : DB00588 decreased stimulated eosinophil adhesion and caused 4-fold increase in annexin 1 expression on the plasma membrane . Inhibition of adhesion by FP was blocked with annexin 1 blocking antibody . Annexin 1 N-terminal mimetic peptide also blocked beta2-integrin adhesion . Translocation of P47712 to the nuclear membrane was significantly blocked by incubation with FP . Blockade was reversed with annexin 1 blocking antibody . CONCLUSION : Blockade of beta2-integrin adhesion by glucocorticoid is regulated by annexin 1 , which blocks P47712 translocation to nuclear membrane .",
      "P10275 YAC transgenic mice recapitulate SBMA motor neuronopathy and implicate VEGF164 in the motor neuron degeneration . X-linked spinal and bulbar muscular atrophy ( SBMA ) is an inherited neuromuscular disorder characterized by lower motor neuron degeneration . SBMA is caused by polyglutamine repeat expansions in the androgen receptor ( AR ) . To determine the basis of AR polyglutamine neurotoxicity , we introduced human AR yeast artificial chromosomes carrying either 20 or 100 CAGs into mouse embryonic stem cells . The AR100 transgenic mice developed a late-onset , gradually progressive neuromuscular phenotype accompanied by motor neuron degeneration , indicating striking recapitulation of the human disease . We then tested the hypothesis that polyglutamine-expanded AR interferes with CREB binding protein ( CBP ) -mediated transcription of vascular endothelial growth factor ( P15692 ) and observed altered CBP-AR binding and P15692 reduction in AR100 mice . We found that mutant AR-induced death of motor neuron-like cells could be rescued by P15692 . Our results suggest that SBMA motor neuronopathy involves altered expression of P15692 , consistent with a role for P15692 as a neurotrophic/survival factor in motor neuron disease .",
      "Expression of vitamin D3 receptor and retinoid receptors in human breast cancer : identification of potential heterodimeric receptors . DB00169 ( VD ) and all-trans-retinoic acid ( DB00755 ) have been postulated as a novel treatment option for breast carcinoma . Since the combined effects of retinoids and VD derivatives are attributed to heterodimeric interactions between members of the nuclear receptor family , the expression patterns of the heterodimers formed by vitamin D3 receptor ( P11473 ) and the retinoid receptors RARs ( P10276 , P10826 and P13631 ) and RXRs ( RXR-alpha , RXR-beta and RXR-gamma ) have been studied by immunohistochemistry in benign and malignant breast tissues . Present results revealed that immunoexpressions to all receptor types studied were higher in both in situ and infiltrative carcinomas than in benign breast diseases . In a variable number of cases of infiltrative carcinoma , immunostaining appeared in the nucleus , whereas in the other two disorders immunostaining was only cytoplasmic . The correlation established between P11473 and the different isoforms of retinoid receptors revealed that P11473 seems to select mainly P10276 to form heterodimers and to exert their properties as transcription factor . The results of this study suggest that this heterodimer plays a critical role in cancer malignancy , and its presence indicates those patient groups presenting a better response to adjuvant therapies based on the combination of vitamin D and DB00755 .",
      "DB00184 induces cell proliferation , invasion and epithelial-mesenchymal transition in a variety of human cancer cell lines . Cigarette smoking is strongly correlated with the onset of nonsmall cell lung cancer ( NSCLC ) . DB00184 , an active component of cigarettes , has been found to induce proliferation of lung cancer cell lines . In addition , nicotine can induce angiogenesis and confer resistance to apoptosis . All these events are mediated through the nicotinic acetylcholine receptors ( nAChRs ) on lung cancer cells . In this study , we demonstrate that nicotine can promote anchorage-independent growth in NSCLCs . In addition , nicotine also induces morphological changes characteristic of a migratory , invasive phenotype in NSCLCs on collagen gel . These morphological changes were similar to those induced by the promigratory growth factor P15692 . The proinvasive effects of nicotine were mediated by alpha7-nAChRs on NSCLCs . RT-PCR analysis showed that the alpha7-nAChRs were also expressed on human breast cancer and pancreatic cancer cell lines . DB00184 was found to promote proliferation and invasion in human breast cancer . The proinvasive effects of nicotine were mediated via a nAChR , Src and calcium-dependent signaling pathway in breast cancer cells . In a similar fashion , nicotine could also induce proliferation and invasion of Aspc1 pancreatic cancer cells . Most importantly , nicotine could induce changes in gene expression consistent with epithelial to mesenchymal transition ( EMT ) , characterized by reduction of epithelial markers like P12830 expression , ZO-1 staining and concomitant increase in levels of mesenchymal proteins like vimentin and fibronectin in human breast and lung cancer cells . Therefore , it is probable that the ability of nicotine to induce invasion and EMT may contribute to the progression of breast and lung cancers .",
      "Identification of an acetylation-dependant P12956 /FLIP complex that regulates FLIP expression and HDAC inhibitor-induced apoptosis . FLIP is a potential anti-cancer therapeutic target that inhibits apoptosis by blocking caspase 8 activation by death receptors . We report a novel interaction between FLIP and the DNA repair protein P12956 that regulates FLIP protein stability by inhibiting its polyubiquitination . Furthermore , we found that the histone deacetylase ( HDAC ) inhibitor DB02546 ( DB02546 ) enhances the acetylation of P12956 , thereby disrupting the FLIP/ P12956 complex and triggering FLIP polyubiquitination and degradation by the proteasome . Using in vitro and in vivo colorectal cancer models , we further demonstrated that DB02546 -induced apoptosis is dependant on FLIP downregulation and caspase 8 activation . In addition , an Q9UBN7 -specific inhibitor Tubacin recapitulated the effects of DB02546 , suggesting that Q9UBN7 is a key regulator of P12956 acetylation and FLIP protein stability . Thus , HDAC inhibitors with anti- Q9UBN7 activity act as efficient post-transcriptional suppressors of FLIP expression and may , therefore , effectively act as ' FLIP inhibitors ' .",
      "DB00107 alleviates the neuroendocrine and cytokine response to bacterial endotoxin in healthy men . DB00107 is a hormone and neurotransmitter found to have anti-inflammatory functions in rodents . Here we used experimental bacterial endotoxinemia to examine the role of exogenous oxytocin administration on innate immune responses in humans . Ten healthy men received , in a randomized , placebo-controlled , crossover design , placebo , oxytocin , LPS , and LPS + oxytocin . DB00107 treatment resulted in a transient or prolonged reduction of endotoxin-induced increases in plasma DB01285 , cortisol , procalcitonin , P01375 , IL-1 receptor antagonist , P05112 , P05231 , macrophage inflammatory protein-1alpha , macrophage inflammatory protein-1beta , monocyte chemoattractant protein-1 ( P13500 ) , interferon-inducible protein 10 , and P15692 . In vitro , oxytocin had no impact on LPS effects in releasing P01375 , P05231 , and P13500 in monocytes and peripheral blood mononuclear cells from healthy human donors . In summary , oxytocin decreases the neuroendocrine and cytokine activation caused by bacterial endotoxin in men , possibly due to the pharmacological modulation of the cholinergic anti-inflammatory pathway . DB00107 might be a candidate for the therapy of inflammatory diseases and conditions associated with high cytokine and P15692 levels .",
      "Modulatory effects of heparin and short-length oligosaccharides of heparin on the metastasis and growth of LMD MDA-MB 231 breast cancer cells in vivo . Expression of the chemokine receptor P61073 allows breast cancer cells to migrate towards specific metastatic target sites which constitutively express P48061 . In this study , we determined whether this interaction could be disrupted using short-chain length heparin oligosaccharides . Radioligand competition binding assays were performed using a range of heparin oligosaccharides to compete with polymeric heparin or heparan sulphate binding to I(125) P48061 . DB01109 dodecasaccharides were found to be the minimal chain length required to efficiently bind P48061 ( 71 % inhibition ; P < 0.001 ) . These oligosaccharides also significantly inhibited P48061 -induced migration of P61073 -expressing LMD MDA-MB 231 breast cancer cells . In addition , heparin dodecasaccharides were found to have less anticoagulant activity than either a smaller quantity of polymeric heparin or a similar amount of the low molecular weight heparin pharmaceutical product , DB06822 . When given subcutaneously in a SCID mouse model of human breast cancer , heparin dodecasaccharides had no effect on the number of lung metastases , but did however inhibit ( P < 0.05 ) tumour growth ( lesion area ) compared to control groups . In contrast , polymeric heparin significantly inhibited both the number ( P < 0.001 ) and area of metastases , suggesting a differing mechanism for the action of polymeric and heparin-derived oligosaccharides in the inhibition of tumour growth and metastases .",
      "Modeling the neurovascular niche : murine strain differences mimic the range of responses to chronic hypoxia in the premature newborn . Preterm birth results in significant cognitive and motor disabilities , but recent evidence suggests that there is variable recovery over time . One possibility that may explain this variable recovery entails variable neurogenic responses in the subventricular zone ( SVZ ) following the period of chronic hypoxia experienced by these neonates . In this report , we have characterized the responses to chronic hypoxia of two mouse strains that represent a wide range of susceptibility to chronic hypoxia . We determined that C57BL/6 pups and neural progenitor cells ( NPCs ) derived from them exhibit a blunted response to hypoxic insult compared with CD-1 pups and NPCs . Specifically , C57BL/6 pups and NPCs exhibited blunted in vivo and in vitro proliferative and increased apoptotic responses to hypoxic insult . Additionally , C57BL/6 NPCs exhibited lower baseline levels and hypoxia-induced levels of selected transcription factors , growth factors , and receptors ( including HIF-1alpha , Q9GZT9 , P23560 , P15692 , P48061 , TrkB , Nrp-1 , P61073 , and NO ) that determine , in part , the responsiveness to chronic hypoxic insult compared with CD-1 pups and NPCs , providing insight into this important and timely problem in perinatology .",
      "4-[3,5-Bis(trimethylsilyl)benzamido] benzoic acid inhibits angiogenesis in colon cancer through reduced expression of vascular endothelial growth factor . 4-[3,5-bis(trimethylsilyl)benzamido] Benzoic acid ( TAC-101 ) has potent antiproliferative , antiangiogenic , and antitumor effects in vitro and in vivo . These effects might be due to TAC-101 binding to retinoic acid receptor alpha ( P10276 ) and interfering with the binding of activator protein-1 ( AP-1 ) to DNA . However , little is known about the detailed mechanism of TAC-101 function . We investigated the mechanism of the antiangiogenic effect of TAC-101 using a rat hepatic metastatic model in vivo and DLD-1 human colon cancer cells in vitro . Liver metastases were induced by portal injection of Q15293 -9 rat colonic cancer cells into F344 rats . TAC-101 ( 8 mg/kg ) was orally administered 5 days per week for 4 weeks and then hepatic tumors were immunohistochemically evaluated for microvessel density ( P53602 ) and vascular endothelial growth factor ( P15692 ) . TAC-101 significantly reduced both P53602 and P15692 expression . Northern blot analysis and ELISA indicated that TAC-101 efficiently inhibited production of P15692 mRNA and protein in DLD-1 cells in a time- and dose-dependent manner . These findings suggest that TAC-101 may inhibit progression and metastasis in colon cancer by interfering with tumor production of P15692 .",
      "A curated database of miRNA mediated feed-forward loops involving MYC as master regulator . BACKGROUND : The MYC transcription factors are known to be involved in the biology of many human cancer types . But little is known about the Myc/microRNAs cooperation in the regulation of genes at the transcriptional and post-transcriptional level . METHODOLOGY/PRINCIPAL FINDINGS : Employing independent databases with experimentally validated data , we identified several mixed microRNA/Transcription Factor Feed-Forward Loops regulated by Myc and characterized completely by experimentally supported regulatory interactions , in human . We then studied the statistical and functional properties of these circuits and discussed in more detail a few interesting examples involving Q01094 , P60484 , P06400 and P15692 . CONCLUSIONS/SIGNIFICANCE : We have assembled and characterized a catalogue of human mixed Transcription Factor/microRNA Feed-Forward Loops , having Myc as master regulator and completely defined by experimentally verified regulatory interactions .",
      "DB09079 , a triple angiokinase inhibitor , enhances cytotoxic therapy response in pancreatic cancer . Angiogenesis remains a sensible target for pancreatic ductal adenocarcinoma ( PDAC ) therapy . P15692 , PDGF , FGF and their receptors are expressed at high levels and correlate with poor prognosis in human PDAC . DB09079 is a triple angiokinase inhibitor that targets P17948 /2/3 , P11362 /2/3 and PDGFRα/β signaling . We investigated the antitumor activity of nintedanib alone or in combination with the cytotoxic agent gemcitabine in experimental PDAC . DB09079 inhibited proliferation of cells from multiple lineages found in PDAC , with gemcitabine enhancing inhibitory effects . DB09079 blocked PI3K/MAPK activity and induced apoptosis in vitro and in vivo . In a heterotopic model , net local tumor growth compared to controls ( 100 % ) was 60.8 ± 10.5 % in the gemcitabine group , -2.1 ± 9.9 % after nintedanib therapy and -12.4 ± 16 % after gemcitabine plus nintedanib therapy . Effects of therapy on intratumoral proliferation , microvessel density and apoptosis corresponded with tumor growth inhibition data . In a PDAC survival model , median animal survival after gemcitabine , nintedanib and gemcitabine plus nintedanib was 25 , 31 and 38 days , respectively , compared to 16 days in controls . The strong antitumor activity of nintedanib in experimental PDAC supports the potential of nintedanib-controlled mechanisms as targets for improved clinical PDAC therapy .",
      "Q00987 is a ubiquitin ligase of P12956 -Akt promotes cell survival by inhibiting Q00987 -dependent P12956 destabilization . Earlier , we have reported that 70 kDa subunit of Ku protein heterodimer ( P12956 ) binds and inhibits Bax activity in the cytosol and that ubiquitin ( Ub ) -dependent proteolysis of cytosolic P12956 facilitates Bax-mediated apoptosis . We found that Q00987 ( human homolog of murine double minute ) has an ability to ubiquitinate P12956 and that Q00987 overexpression in cultured cells causes a decrease in P12956 expression levels . An interaction between P12956 and Q00987 was shown by means of immunoprecipitation , whereas none could be shown between 80 kDa subunit of Ku protein heterodimer and Q00987 . Vascular endothelial growth factor ( P15692 ) is known to inhibit endothelial cell ( EC ) apoptosis through an Akt-mediated survival kinase signal ; however , the mechanism underlying this inhibition of apoptosis has not been fully elucidated . We found that P15692 inhibited cytosolic P12956 degradation induced by apoptotic stress . It is known that Akt-dependent phosphorylation of Q00987 causes nuclear translocation of Q00987 followed by Q00987 -mediated inactivation of p53 . We found that P15692 stimulated nuclear translocation of Q00987 in EC and efficiently inhibited P12956 degradation . We also found that constitutively active Akt , but not kinase-dead Akt , inhibited P12956 degradation in the cytosol . Furthermore , P12956 knockdown diminished antiapoptotic activity of Akt . Taken together , we propose that Q00987 is a P12956 Ub ligase and that Akt inhibits Bax-mediated apoptosis , at least in part , by maintaining P12956 levels through the promotion of Q00987 nuclear translocation .",
      "Pharmacological properties of thalidomide and its analogues . Thalidomide and its immunomodulatory imide drugs ( IMiDs ) analogues DB00480 ( DB00480 , DB00480 ) and CC-4047 ( Actimid , DB08910 ) have been used as anti-inflammatory and anticancerous drugs in the recent years . Thalidomide and IMiDs inhibit the cytokines tumour necrosis factor-alpha ( P01375 ) , interleukins ( IL ) 1-beta , 6 , 12 , and granulocyte macrophage-colony stimulating factor ( GM- P04141 ) . They also costimulate primary human T , NKT and NK lymphocytes inducing their proliferation , cytokine production , and cytotoxic activity . On the other hand , the compounds are anti-angiogenic , anti-proliferative , and pro-apoptotic . Thalidomide analogues have been used as inhibitors of alpha glucosidase and could be potential drugs for diabetes treatment . In this review , we explore the current trend of the different structures , the new patents , and the possible new applications in different pathologies .",
      "Loss of epigenetic Kruppel-like factor 4 histone deacetylase ( KLF-4-HDAC ) -mediated transcriptional suppression is crucial in increasing vascular endothelial growth factor ( P15692 ) expression in breast cancer . Vascular endothelial growth factor ( P15692 ) is recognized as an important angiogenic factor that promotes angiogenesis in a series of pathological conditions , including cancer , inflammation , and ischemic disorders . We have recently shown that the inflammatory transcription factor P56270 is , at least in part , responsible for the marked increase of P15692 levels in breast cancer . Here , we show that P56270 -mediated induction of P15692 is repressed by KLF-4 transcription factor . KLF-4 is abundantly present in normal breast epithelial cells , but its level is considerably reduced in breast cancer cells and clinical cancer tissues . In the human P15692 promoter , P56270 - and KLF-4-binding elements are overlapping , whereas P56270 induces and KLF-4 suppresses P15692 expression . Ectopic overexpression of KLF-4 and RNAi-mediated inhibition of endogenous KLF-4 supported the role of KLF-4 as a transcriptional repressor of P15692 and an inhibitor of angiogenesis in breast cancer cells . We show that KLF-4 recruits histone deacetylases ( HDACs ) -2 and -3 at the P15692 promoter . Chronological ChIP assays demonstrated the occupancy of KLF-4 , Q92769 , and O15379 in the P15692 promoter in normal MCF-10A cells but not in MDA-MB-231 cancer cells . Co-transfection of KLF-4 and HDAC expression plasmids in breast cancer cells results in synergistic repression of P15692 expression and inhibition of angiogenic potential of these carcinoma cells . Together these results identify a new mechanism of P15692 up-regulation in cancer that involves concomitant loss of KLF-4-HDAC-mediated transcriptional repression and active recruitment of P56270 -mediated transcriptional activation .",
      "Gender difference in the activity but not expression of estrogen receptors alpha and beta in human lung adenocarcinoma cells . The higher frequency of lung adenocarcinoma in women smokers than in men smokers suggests a role for gender-dependent factors in the etiology of lung cancer . We evaluated estrogen receptor ( ER ) alpha and beta expression and activity in human lung adenocarcinoma cell lines and normal lung fibroblasts . Q8N1N2 -length ERalpha and ERbeta proteins were expressed in all cell lines with higher ERbeta than ERalpha . Although estradiol ( E(2) ) binding was similar , E(2) stimulated proliferation only in cells from females , and this response was inhibited by anti-estrogens 4-hydroxytamoxifen ( DB04468 ) and DB00947 . In contrast , E(2) did not stimulate replication of lung adenocarcinoma cells from males and DB04468 or ICI did not block cell proliferation . Similarly , transcription of an estrogen response element-driven reporter gene was stimulated by E(2) in lung adenocarcinoma cells from females , but not males . P06401 ( PR ) expression was increased by E(2) in two out of five adenocarcinoma cell lines from females , but none from males . E(2) decreased P12830 protein expression in some of the cell lines from females , as it did in MCF-7 breast cancer cells , but not in the cell lines from males . Thus , ERalpha and ERbeta expression does not correlate with the effect of ER ligands on cellular activities in lung adenocarcinoma cells . On the other hand , coactivator Q15648 expression was higher in lung adenocarcinoma cells from females versus males and higher in adenocarcinoma cells than in normal human bronchial epithelial cells . Q15648 and other ER coregulators may contribute to differences in estrogen responsiveness between lung adenocarcinoma cells in females and males .",
      "Stimulation of epithelial repair is a likely mechanism for the action of mifepristone in reducing duration of bleeding in users of progestogen-only contraceptives . Many women using progestogen ( P ) -only contraceptives experience uterine bleeding problems . In clinical trials , a single low dose of mifepristone , given to DB00294 users at the beginning of a bleeding episode reduced the number of bleeding days by approximately 50 % compared with controls . In this study , a single dose of mifepristone was administered to etonogestrel ( P17813 ) -exposed pseudo-pregnant mice , 5 days after artificial decidualization was induced when the endometrium showed signs of bleeding . Control mice received vehicle alone . Mice were culled 12- , 18- , 24- and 48-h post-treatment . In the continued presence of P17813 , a single dose of mifepristone stimulated tissue breakdown followed by very rapid repair : most treated tissues were fully restored to the pre-decidualized state by 48 h post-treatment . During repair , proliferating cells ( Ki67 immunostained ) were localized to a band of cells around the basal area in breaking down tissues and to the repairing luminal epithelium and glands . P06401 -positive cells were largely localized to the basal area of the breaking down tissue in treated mice compared with decidual cells in controls . Oestrogen receptor-positive cells were observed in the repairing luminal epithelium and glands compared with the decidua and the basal region in control tissues . It is concluded that mifepristone treatment stimulates rapid restoration of luminal epithelial integrity : such action may be a key event in reducing the number of bleeding days observed in women using DB00294 who were treated with a single dose of mifepristone .",
      "17 DB00783 -mediated growth inhibition of MDA-MB-468 cells stably transfected with the estrogen receptor : cell cycle effects . P03372 ( ER ) -negative MDA-MB-468 human breast cancer cells were stably transfected with wild-type human ER and utilized as a model for investigating estrogen- and aryl hydrocarbon ( Ah ) -responsiveness . Treatment of the stably transfected cells with 10 nM 17 beta-estradiol ( E2 ) resulted in a significant inhibition ( > 60 % ) of cell proliferation and DNA synthesis , which was blocked by 10(-7) M ICI 182 780 . Analysis by flow cytometry indicated that treatment with E2 increased the percentage of cells in G0/ P55008 ( from 68.8 to 89.4 ) and decreased cells in S ( from 18.4 to 3.4 ) and G2/M ( from 12.8 to 7.2 ) phases of the cell cycle . The effects of E2 on the major cyclins , cyclin-dependent kinases and cyclin-dependent kinase inhibitors , retinoblastoma protein ( RB ) , Q01094 , and cyclin-dependent kinase activities were also investigated in the stably transfected MDA-MB-468 cells . The results demonstrated that the growth inhibitory effects of 10(-8) M E2 in ER stably transfected MDA-MB-468 cells were associated with modulation of several factors required for cell cycle progression and DNA synthesis , including significant induction of the cyclin-dependent kinase inhibitor p21cip-1 ( > 4-fold increase after 12 h ) and decreased Q01094 and P12004 protein levels . These results show that the growth-inhibitory effects of E2 in the stably transfected cells were due to multiple factors which result in growth arrest in G0/ P55008 and inhibition of DNA synthesis ."
   ],
   "query":"interacts_with DB09079?"
}
```